.circleci/set_env_pact.sh

```diff
@@ -6,25 +6,28 @@ PACT_BROKER_USER="admin"
 PACT_BROKER_URL="pact-broker.api.opg.service.justice.gov.uk"
 API_VERSION="v1"
 
-export SECRETSTRING=$(aws sts assume-role \
---role-arn "arn:aws:iam::${ACCOUNT}:role/get-pact-secret-production" \
---role-session-name AWSCLI-Session | \
-jq -r '.Credentials.SessionToken + " " + .Credentials.SecretAccessKey + " " + .Credentials.AccessKeyId' 2>/dev/null)
+if [ $(aws --version 2>&1 | grep -c "aws-cli") -ne 0 ]
```

Comment:
Is this just checking aws or aws-cli is installed?

Reply:
if you do aws --version, then you get back a string like 'aws-cli/1.17.12 Python/2.7.16 Darwin/17.7.0 botocore/1.14.12' when it's the pip installed aws-cli. :)